utility/types/account.go

```diff
@@ -0,0 +1,16 @@
+package types
+
+// TODO: Consider refactoring PoolNames and statuses to an enum
+// with appropriate enum <-> string mappers where appropriate.
+// This can make it easier to track all the different states
+// available.
+const (
+	ServiceNodeStakePoolName = "SERVICE_NODE_STAKE_POOL"
```

Comment:
Can this be an enum?

Reply:
Yes it could be

Reply:
Can you add a TODO or change it in this PR? I think it'll make some things simpler to understand.